Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/BoomBox/README.md

```diff
@@ -0,0 +1,9 @@
+# Boom Box
+## Screenshot
+
+![screenshot](screenshot/screenshot.jpg)
+
+## License Information
```

Comment:
Can we create models of our own, Perhaps using Paint3D, to avoid additional licensing?

Reply:
What motivations are there for not reusing this?

Reply:
it's a dependency that requires license registration for us to ship. if we create our own test models, there is no extra license validation step before MRTK ships

Reply:
I'm not sure we have to register things that are public domain.

Reply:
And being provided by Microsoft likely helps. I can follow up on this.

Reply:
Thanks @keveleigh let us know.